src/lib/es.js

```diff
@@ -366,9 +366,8 @@ async function constructSearchParams(parameters, page, limit) {
     body.search_after = buildSearchAfter(parameters)
   }
 
-  // Specifying the scroll parameter makes the total work
   const searchParams = {
-    index: collections || '*,-*kibana*,-collections',
+    index: collections || '*,-.*,-collections',
```

Comment:
`-.*` excludes all the system indices, including ones named `.kibana*`, `.opendistro*` and `.opensearch*`